content/geometry/object-human/human-model-structure.adoc

```diff
@@ -2,1213 +2,997 @@
-                |_ [[../geometry/object-classes.html#_lower_thumb_left_t Lower_Thumb_Left (T)]]
-            |_ [[../geometry/object-classes.html#_gripping_point_left_t Gripping_Point_Left (T)]]
-      |_ [[../geometry/object-classes.html#_shoulder_right_t Shoulder_Right (T)]]
-        |_ [[../geometry/object-classes.html#_upper_arm_right_t Upper_Arm_Right (T)]]
-          |_ [[../geometry/object-classes.html#_lower_arm_right_t Lower_Arm_Right (T)]]
-            |_ [[../geometry/object-classes.html#_full_hand_right_t Full_Hand_Right (T)]]
-            |_ [[../geometry/object-classes.html#_hand_right_t Hand_Right (T)]]
-              |_ [[../geometry/object-classes.html#_hand_right_t Upper_Fingers_Right (T)]]
-                |_ [[../geometry/object-classes.html#_lower_fingers_right_t Lower_Fingers_Right (T)]]
-              |_ [[../geometry/object-classes.html#_upper_thumb_right_t Upper_Thumb_Right (T)]]
-                |_ [[../geometry/object-classes.html#_lower_thumb_right_t Lower_Thumb_Right (T)]]
-            |_ [[../geometry/object-classes.html#_gripping_point_right_t Gripping_Point_Right (T)]]      
-    |_ [[../geometry/object-classes.html#_sitting_point_t Sitting_Point (T)]]
-    |_ [[../geometry/object-classes.html#_upper_leg_right_t Upper_Leg_Right (T)]]
-      |_ [[../geometry/object-classes.html#_lower_leg_right_t Lower_Leg_Right (T)]]
-        |_ [[../geometry/object-classes.html#_full_foot_right_t Full_Foot_Right (T)]]
-        |_ [[../geometry/object-classes.html#_standing_point_right_t Standing_Point_Right (T)]]
-    |_ [[../geometry/object-classes.html#_upper_leg_left_t Upper_Leg_Left (T)]]
-      |_ [[../geometry/object-classes.html#_lower_leg_left_t Lower_Leg_Left (T)]]
-        |_ [[../geometry/object-classes.html#_full_foot_left_t Full_Foot_Left (T)]]
-        |_ [[../geometry/object-classes.html#_standing_point_left_t Standing_Point_Left (T)]]
-|_ [[../geometry/object-classes.html#_body Body]]
-|_ [[../geometry/object-classes.html#_clothing Clothing]]
-|_ [[../geometry/object-classes.html#_hair Hair]]
-end legend
+|Grp_Root
+  |Skeleton
+    |Root
+      |Lower_Spine
+        |Hip_Left
+        |Hip_Right
+        |Middle_Spine
+          |Upper_Spine
+            |Neck
+              |Head
+                |Eye_Left
+                |Eye_Right
+          |Shoulder_Left
+            |Upper_Arm_Left
+              |Lower_Arm_Left
+                |Hand_Left
+                  |Full_Fingers_Left
+                  |Full_Thumb_Left
+            |Upper_Arm_Right
+              |Lower_Arm_Right
+                |Hand_Right
+                  |Full_Fingers_Right
+                  |Full_Thumb_Right
+      |Upper_Leg_Left
+        |Lower_Leg_Left
+          |Foot_Left
+            |Full_Toes_Left
+      |Upper_Leg_Right
+        |Lower_Leg_Right
+          |Foot_Right
+            |Full_Toes_Right
+  |Grp_Accessories
+    |Accessorie
+  |Grp_Body
+    |Human
+  |Grp_Clothing
+    |Vest
+  |Grp_Hair
+    |Hair
```

Comment:
Links to the sections are missing here. @FKlopfer you assist in adding them back in?

Reply:
Added links to structure overview. Some chapters exist only in the overview yet, they need to be created: Skeleton, Human, Vest, Hair, Accessorie.

The order of the chapters also needs to be adjusted when everything is finished.